### PR TITLE
Adding 'KUBECTL_SKIP_DEPLOY' check and skip

### PR DESCRIPTION
**Pull request by PhillypHenning**

### CODE REVIEW

**Comment by LeoDiazL** [comment]
Code looks good.

File: deploy.sh [hunk 1]
```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 set -ex
 
+if [ "$SKIP_DEPLOY_KUBECTL" == "true" ] || [ "$KUBECTL_SKIP_DEPLOY" == "true" ]; then
+  echo "KUBECTL_SKIP_DEPLOY is set.  Skipping."
+  exit 0
```
Comment (arm4b) [inline deploy.sh:6]: I see the `kubectl` doesn't have any schema like in the other plugins. Why is that?

How would this setting work then for Ansible plugin:  
```yaml
  options:
    skip-deploy: false
```

what's the flow?

File: deploy.sh [hunk 2]
```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 set -ex
 
+if [ "$SKIP_DEPLOY_KUBECTL" == "true" ] || [ "$KUBECTL_SKIP_DEPLOY" == "true" ]; then
```
Comment (arm4b) [inline deploy.sh]: Why do we need two ENV vars to control this behavior instead of one?

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks like the plugin README needs an update to document this new ENV var so it's not lost in the code.

Also curious about the flow here when using `skip-deploy` vs `KUBECTL_SKIP_DEPLOY`.

**Review by arm4b — APPROVED** [review]

**Pull request merged**